packages/nocodb/src/helpers/responseHelpers.ts

```
@@ -0,0 +1,14 @@
+export function maskKnexConfig(payload: Partial<{ config: any }>) {
+  /*
+    remove password from knex config
+  */
+  if (
+    payload.config &&
+    payload.config.connection &&
+    payload.config.connection.password
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)